{"owner": {"display_name": "Zach", "reputation": 624, "profile_image": "https://www.gravatar.com/avatar/cdbe9e4e60271f3ff3729b382e45e6a6?s=128&d=identicon&r=PG", "user_id": 1767263, "user_type": "registered", "accept_rate": 69, "link": "http://stackoverflow.com/users/1767263/zach"}, "link": "http://stackoverflow.com/questions/30389771/importerror-no-module-named-django-core-management-when-using-manage-py", "tags": ["python", "django", "python-2.7", "importerror", "django-manage.py"], "creation_date": 1432276137, "score": 14, "is_answered": true, "view_count": 23206, "last_activity_date": 1477720707, "question_id": 30389771, "accepted_answer_id": 30409085, "answer_count": 9, "title": "ImportError: No module named django.core.management when using manage.py"}